### ISSUE TITLE
[BUG]: DJbox (Japan) - SCPS-15082 - 56275BE9

### ISSUE THREAD

**Issue by spacemul** [PCSX2/pcsx2#5169]
### Describe the Bug

Just black screen at start

### Reproduction Steps

Just launch the game

### PCSX2 Revision

1.7.2120

### Operating System

Windows 10 (64bit)

### CPU

Ryzen 5 3600

### GPU

RTX 3070

**Comment by gilderoylockhart01** [comment]
Same as the other thread, with a blockdump, we can see what is wrong with the game.

**Comment by spacemul** [comment]
[DJbox (Japan) (2021-11-16 20-21-56).zip](https://github.com/PCSX2/pcsx2/files/7729898/DJbox.Japan.2021-11-16.20-21-56.zip)

**Comment by gilderoylockhart01** [comment]
Thanks, actually I run out of block if I enable the timing hack or the EE interpreter with the cache enabled, does those options work for you?

**Comment by spacemul** [comment]
I tried these 2 options but only black screen here on 1.7.2133

**Comment by Goatman13** [comment]
Game unpack/decrypt EE overlays with use of FPU. That result in corrupted mips code overlays on pcsx2. This has been tested on different emulator with softfloats available and function that generate inaccuracy is located at 0x128268. Since pcsx2 add/sub is accurate and function don't use div/sqrt/rsqrt, this is most likely mul inaccuracy. Game was additionally tested with old fpu mul mantisa hack, but it is not enough to fix it.

Just in case, it doesn't seems to be clamaping issue.

**Reproduction Steps**
Launch game. Check unpacked code at 0x2821E0, 00282714, etc. This isn't only corruption that happen of course. To additionally confirm issue you can check the same code when game is launched with nearest rounding mode, code is completely broken then (initial jump at 0x100E24).

**Comment by refractionpcsx2** [comment]
Yeah I think kr_ps2 looked at this with me and said it did a while decryption routine but pcsx2 is just slightly out a bunch of times which ruins it

**Comment by Sukotto-1999** [comment]
Here's my chart of getting DJBox to work on PCSX2:

DJBox is an MP3 mixing game on the PS2. You use the dualshock 2 to mix the sounds.

Here's my chart, the game requires the PSBBN like i mentioned to get it to work. I tried using the HDD FIX on real hardware but no help unless i get PSBBN to work eventually.

Task list for the developers to complete (except for MagicGate)
- [x] Make a complete overhaul on the FPU emulation so EE overlays can be decrypted succesfully and to fix issues in FPU-dependant games. (Driv3r now has workarounds with a specific fork of PCSX2.)
- [ ] Improve TLB emulation so it can boot PSBBN if dump is decrypted and configure the network settings (and access the things on someone's hacked BBN dump like [this](https://youtu.be/HF8vxL_tziQ?t=60).)
- [ ] MagicGate emulation (to fix the TLB miss for Sony Utility Discs without a hacked version) (EDIT: A tool known as PS3MCA and most specifically mecha_emu.c implementing MagicGate was subject to being taken down by Sony. This now strictly goes into a private unofficial fork, again to try and minimize cases like this, unless we get some legal advice or permission from Sony and other subsidiaries.)
- [ ] Proper network/HDD LAN (dev9) emulation, to fix network errors.

These are the improvements needed to boot the game and access things that are mentioned in the [official trailer of the game](https://www.youtube.com/watch?v=CQytIBK3xPc).

**Comment by Sukotto-1999** [comment]
@refractionpcsx2 I found that Sony sent a [DMCA takedown against PS3MCA and more specifically mecha_emu.c implementing Magicgate](https://github.com/github/dmca/blob/master/2011/2011-06-21-sony.markdown). That means . . .  we need decryption instead of implementing MagicGate and MagicGate support now goes strictly into a private fork of the PCSX2 emulator. Was this DMCA the reason why we never implement this crap?

**Comment by Sukotto-1999** [comment]
Hello everyone. I'm just commenting here that due to strict rules against illegal acts I am going to STAY away from the PCSX2 Team and PCSX2 Discord and from being a member. I will and i mean, i will ONLY be a game tester. Period.

**Comment by F0bes** [comment]
> Hello everyone. I'm just commenting here that due to strict rules against illegal acts I am going to STAY away from the PCSX2 Team and PCSX2 Discord and from being a member. I will and i mean, i will ONLY be a game tester. Period.

You're going to stay away due to our anti piracy policies? No one cares. Bye.

**Comment by electronicsignals** [comment]
Any news on this game? Would love to help get this one working if possible. Still freezing black screen after PlayStation logo as of  March 2026 with v2.7.166

**Comment by JordanTheToaster** [comment]
> Any news on this game? Would love to help get this one working if possible. Still freezing black screen after PlayStation logo as of March 2026 with v2.7.166

No timeline currently on when it will be working.
